utils/run_test.sh

```diff
@@ -94,24 +94,26 @@ if [ -d "/mnt/daos" ]; then
     run_test build/src/common/tests/sched
     run_test build/src/common/tests/drpc_tests
     run_test build/src/client/api/tests/eq_tests
-	run_test build/src/bio/smd/tests/smd_ut
+    run_test build/src/bio/smd/tests/smd_ut
     run_test src/vos/tests/evt_ctl.sh
     run_test src/vos/tests/evt_ctl.sh pmem
     run_test "${SL_PREFIX}/bin/vea_ut"
     run_test src/rdb/raft_tests/raft_tests.py
-    # Satisfy CGO requirements for go-spdk binding and internal daos imports
-    LD_LIBRARY_PATH="${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
-    export LD_LIBRARY_PATH="${SL_PREFIX}/lib64:${LD_LIBRARY_PATH}"
-
+    # Satisfy CGO requirements for go-spdk bindings
     export CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib -L${SL_PREFIX}/lib64"
     export CGO_CFLAGS="-I${SL_SPDK_PREFIX}/include"
     run_test src/control/run_go_tests.sh
-    # Environment variables specific to the rdb tests
-    export PATH="${SL_PREFIX}/bin:${PATH}"
-    # Satisfy requirement for starting daos_server w/o config file
-    export CRT_PHY_ADDR_STR=ofi+sockets
-    export OFI_INTERFACE=lo
+    unset CGO_CFLAGS CGO_LDFLAGS
+    # Satisfy internal daos imports & runtime daos_server deps
+    OLD_LDLIBPATH=${LD_LIBRARY_PATH}
+    LD_LIBRARY_PATH="${SL_PREFIX}/lib64:${OLD_LDLIBPATH}"
+    LD_LIBRARY_PATH="${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
+    OLD_PATH=${PATH}
+    PATH="${SL_PREFIX}/bin:${OLD_PATH}"
+    export LD_LIBRARY_PATH PATH
     run_test src/rdb/tests/rdb_test_runner.py "${SL_OMPI_PREFIX}"
+    export PATH=${OLD_PATH} LD_LIBRARY_PATH=${OLD_LDLIBPATH}
```

Comment:
I don't know if maybe it would make more sense to just run that rdb test last, rather than having to set and reset the environment.